addons/web/static/src/core/l10n/localization_service.js

```diff
@@ -54,7 +54,10 @@ export const localizationService = {
             }
         }
 
-        Object.setPrototypeOf(translatedTerms, terms);
+        // Object.setPrototypeOf(translatedTerms, terms);
+        Object.assign(translatedTerms, terms);
+        translatedTerms.__isLoaded__ = true;
+
         env._t = _t;
```

Comment:
Now that `_t` handles both lazy and “regular” loading, I think you should remove this and put the actual `_t` function directly into `makeEnv`

Reply:
We've been debating not putting _t in the env at all anymore and always importing it. I think the original reason to put it in the env was to allow it being used in xml templates but we can't extract terms from js expressions in owl templates anyway so that point is moot.